src/current/v25.4/logging-use-cases.md

```diff
@@ -269,6 +269,67 @@ I210323 18:50:04.518707 1182 8@util/log/event_log.go:32 ⋮ [n1,client=‹[::1]:
 All possible `SENSITIVE_ACCESS` event types are detailed in the [reference documentation]({% link {{ page.version.version }}/eventlog.md %}#sql-access-audit-events). For a detailed tutorial on table auditing, see [SQL Audit Logging]({% link {{ page.version.version }}/sql-audit-logging.md %}).
 {{site.data.alerts.end}}
 
+#### Example: Unsafe internals
+
+{{site.data.alerts.callout_danger}}
+In future releases, the [`allow_unsafe_internals` session variable]({% link {{ page.version.version }}/session-variables.md %}#allow-unsafe-internals) will default to `off` and then will be removed. To assess potential downstream impacts on your setup, set `allow_unsafe_internals` to `off` in a non-production environment.
```

Comment:
Do we have details on how they can asses documented somewhere and how it can impact? For instance, calling out that monitoring tools or scripts that could be accessing these internals will be impacted